### PR TITLE
added option to sort torrent links retrieved by number of seeders

### PR DESCRIPTION
**Pull request by DrewMcCarthy**
replaced references to deprecated tvdb_exceptions with tvdb_api

### CODE REVIEW

File: torrentleech_monitor/settings.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 # General settings.
 
 # If True, E-Mail reports will be sent.
-SHOULD_SEND_REPORT = True
+SHOULD_SEND_REPORT = False
```
Comment (ofir123) [inline torrentleech_monitor/settings.py]: Why not `True`?

File: torrentleech_monitor/monitor.py [hunk 2]
```diff
@@ -90,6 +93,9 @@ def _get_torrents(show_name, season_number, episode_number, session):
                 results_list = [t.find('a')['href'] for t in table.find_all('td', 'quickdownload')]
                 sizes_list = [t.string for t in table.find_all('td') if t.string and
                               ('GB' in t.string or 'MB' in t.string)]
+                seeders_list = [t.get_text() for t in table.find_all('td', 'seeders')]
```
Comment (ofir123) [inline torrentleech_monitor/monitor.py]: Use `int` conversion here (instead of `map` in `sort_by_seeders`). 
No use in keeping this information as str..

**Review by ofir123 — CHANGES_REQUESTED** [review]
Great addition!
2 minor notes and then I'll merge...

File: torrentleech_monitor/settings.py (hunk 1 above)
Comment (DrewMcCarthy) [reply torrentleech_monitor/settings.py]: Sorry just my setting that i forgot to revert

**Review by DrewMcCarthy — COMMENTED** [review]
updated per comments

**Review by ofir123 — APPROVED** [review]

**Pull request merged**